FlowCrypt/Controllers/Setup/SetupGenerateKeyViewController.swift

```diff
@@ -119,10 +113,9 @@ private actor Service {
             try appContext.passPhraseService.savePassPhrase(with: passPhrase, storageMethod: .memory)
         }
 
-        await submitKeyToAttesterAndShowAlertOnFailure(
+        try await submitKeyToAttester(
```

Comment:
In case this operation is failed pass phrase and key are already saved in app. 
Should we change order of this operation during generating backup?

Reply:
@tomholub @sosnovsky

Reply:
please check implementation for 
```
func setupAccount(
        passPhrase: String,
        storageMethod: StorageMethod,
        viewController: ViewController
) 
```

Reply:
<img src="https://user-images.githubusercontent.com/13693255/149664196-8c33a4d9-e91d-43e5-a977-6e3c333054d8.png" width="300"/>

Reply:
@tomholub can you please help with this error?

Reply:
@sosnovsky Do you aware of what this error message means?

Reply:
Not sure, but looks like something wrong with checking your existing public key. As error says you already have `previously recorded key`, but app tries to submit another one.
In this case app should open `SetupManuallyImportKeyViewController` instead of `SetupGenerateKeyViewController`.
Does it work well for you on `master` branch or it fails there too?

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/pull/1311#discussion_r787679362 
Looks like due to this issue we failed to backup key in inbox but it was submitted to attester

Reply:
The error rendering is clear now, and for your manual testing, using https://flowcrypt.com/attester/update-public-keys?intention=delete will help you as @sosnovsky mentioned.

As far as user experience is concerned, when this error happens, they are screwed because they won't know what to do, but that's ok for now since few people if anyone uses IMAP/SMTP.